march_gait_selection/CMakeLists.txt

```
@@ -2,31 +2,30 @@ cmake_minimum_required(VERSION 2.8.3)
 project(march_gait_selection)
 
 find_package(catkin REQUIRED COMPONENTS
+    march_description
     march_shared_classes
     march_shared_resources
     std_msgs
 )
 
 catkin_python_setup()
 
-catkin_package(
-    CATKIN_DEPENDS
+catkin_package(CATKIN_DEPENDS
+    march_description
     march_shared_classes
     march_shared_resources
     std_msgs
 )
 
-catkin_install_python(PROGRAMS scripts/${PROJECT_NAME}_node
+catkin_install_python(PROGRAMS scripts/${PROJECT_NAME}_node test/run_tests
     DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
-install(DIRECTORY launch
+
+install(DIRECTORY launch test/launch
```

It is not necessary to install these launch files and also you do not want users accidentally launching tests instead of the node
```suggestion
install(DIRECTORY launch
```